runner/src/server/plugins/engine/pageControllers/RepeatingFieldPageController.ts

```diff
@@ -151,11 +155,7 @@ export class RepeatingFieldPageController extends PageController {
   addRowsToViewContext(response, state) {
     let rows = {};
     if (this.options!.summaryDisplayMode!.samePage) {
-      if (state.MultiInputField) {
```

Comment:
This is a fix to an issue i caught with multiinput that i have put in as part of this PR. If the component was named anything other than MultiInputField it would cause display issues as well.